tycho-compiler-plugin/src/main/java/org/eclipse/tycho/compiler/AbstractOsgiCompilerMojo.java

```diff
@@ -385,29 +386,55 @@ private void doCompile() throws MojoExecutionException, MojoFailureException {
             super.execute();
             doCopyResources();
         }
-        //Check for MR JAR compile
-        OsgiManifest manifest = bundleReader.loadManifest(project.getBasedir());
-        if (Boolean.parseBoolean(manifest.getValue("Multi-Release"))) {
```

Comment:
According to the spec the manifest header MUST be set so it won't work otherwise (e.g. in IDEs), also we do not want take extra work for projects that have not explicitly enabled it (like the same goal in the MR-JAR spec!)

As mentioned above, this check should be remain! and then we should only check for the classpath attributes first inside the if-block.